WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/StatsUtils.kt

```diff
@@ -173,7 +173,7 @@ class StatsUtils
         entries: List<Line>
     ): List<String> {
         val contentDescriptions = mutableListOf<String>()
-        entries.forEachIndexed { index, bar ->
+        entries.forEachIndexed { _, bar ->
```

Comment:
@ParaskP7 - all is good with the change you made ... just questioning why the use of `forEachIndexed` without actually using the index. 🤷 What am I missing? And I don't expect a change for this, it's not part of the PR.

Reply:
Ha, yea, you are right, this could have further improved to become a plain `entries.forEach { bar -> ... }`, the indexed version is not needed as the `index` ends up becoming unused. I am seeing this `entries.forEachIndexed { index, bar -> ... }` being actually used within `getBarChartEntryContentDescriptions(...)` above, and as such, I think this was just a copy-pasting overside at that point to create the `getLineChartEntryContentDescriptions(...)` below, which this time didn't have any need for that.

Reply:
PS: I am going to fix that on my next PR. It would be just another refactoring commit on top of all the others commits! 🎉 Thanks for spotting it! 🙇 ❤️ 🌟